src/components/SourcePicker.js

```diff
@@ -67,7 +67,7 @@ class SourcePicker extends Component {
     const { activeProvider, selectedComponent } = this.state
     const { value } = this.props
     return (
-      <Grid className="main-container">
+      <Grid className="main-container" id="source-picker">
```

Comment:
this feels natural (id is the id, classnames are actual styles)

Reply:
Since the component is really unique over the application, that's the correct approach.